src/css/announcement-email.css

```diff
@@ -0,0 +1,107 @@
+.announcement-email {
+  background-color: var(--redpanda-900);
+  color: white;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  font-size: 21px;
+  overflow: hidden;
+  padding: 15px;
+  transition: opacity 0.4s ease-in-out;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 999;
+  opacity: 0;
+  max-height: 8rem;
+}
+
+.emailInput {
```

Comment:
Put the rest of this CSS in the same scope:

For example:

```
.announcement-email .emailInput {
  ...
}
```

Reply:
This should only be done with the rules that overlap, right?

Reply:
@Deflaimun  it's best practice to scope the selectors in case we ever create other elements with the same class by accident

Reply:
We keep each scope in a separate file for this reason